gcs/src/gcs_gcomm.cpp

```diff
@@ -223,10 +222,6 @@ class GCommConn : public Toplay
         {
             mutex_.unlock();
         }
-||||||| 63116854
-=======
-        delete tp_;
```

Comment:
We don't need this delete? According to the upstream commit this fixes a memory leak.

Reply:
yes, this delete is at the beginning of the method